docs/core/compatibility/toc.yml

```diff
@@ -58,6 +58,10 @@ items:
             items:
               - name: Deprecated desktop Windows/macOS/Linux MonoVM runtime packages
                 href: deployment/9.0/monovm-packages.md
+          - name: Extensions
```

Comment:
This API seems more like core .NET libraries to me than extensions?

Reply:
I'm not sure of the correct terminology here. Generally, any library that extends `EnumConverter `is impacted by this change, for example [TypeUnitConverter](https://github.com/dotnet/winforms/blob/d223c9e1eb6b46b3be6ca0652b0d73b0820eb517/src/System.Drawing.Common/src/System/Drawing/FontConverter.cs#L471).

Reply:
Not extensions, core libraries.  EnumConverter is in the shared framework.